### PR TITLE
Basic YouTube Player component initialiased referring to issue #78

### PR DESCRIPTION
**Pull request by sarru1291**
**Description**

> What does this PR achieve? Is it a feature/hotfix/refactor? Does it close an Issue? If so, use `Closes #`.
This PR is for the review to get the flag to continue further on YouTube Wrapper component. It contains the first basic youtube player component using iframe player API.

### CODE REVIEW

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { Component} from 'react'
```
Comment (iisa) [inline packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js]: let's move this file into the `/sandbox/theatres/components/audio_player/players_by_type` directory

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  }
+  onPlayerReady(e) {
+    e.target.playVideo();
+    // alert('video is on...');
+  }
+ 
+  render () {
+    return (
+      <div className='YoutubePlayer'>
+        <div ref={(r) => { this.state.playerAnchor = r }}></div>
```
Comment (iisa) [inline packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js]: Suggestion: the ref has already been made, use it directly as the ref, not an anonymous function

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      </div>
+    )
+  }
+}
+
+YoutubePlayer.propTypes = {
+  videoId: PropTypes.string.required,
```
Comment (iisa) [inline packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js]: Blocker: we need it to take in an array of videos

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, number } from '@storybook/addon-knobs';
+import YoutubePlayerTestComponent from './YoutubePlayer';
+
+storiesOf('Sandbox', module)
```
Comment (iisa) [inline packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js]: adding this is great!

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+          })
+        }
+  }
+  loadAPI(){
+    const tag = document.createElement('script')
+    tag.src = 'https://www.youtube.com/iframe_api'
+    const firstScriptTag = document.getElementsByTagName('script')[0]
+    firstScriptTag.parentNode.insertBefore(tag, firstScriptTag)
+  }
+  onPlayerReady(e) {
```
Comment (iisa) [inline packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js]: Blocker: we need to tap into more of the API functionality. Please check original issue for the events that we want to handle with this wrapper.

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js (hunk 4 above)
Comment (iisa) [reply packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js]: let's create a new sandbox section just for `audio players!

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js (hunk 3 above)
Comment (sarru1291) [reply packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.js]: ya sure, it was taken single just as a demo.

File: packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js (hunk 4 above)
Comment (Batey96) [reply packages/ia-components/sandbox/youtube-wrapper/youtube-player/YoutubePlayer.stories.js]: > let's create a new sandbox section just for `audio players!

okay working on it

**Pull request closed without merging**